Dispatcher/Controllers/HomeViewController.swift

```diff
@@ -27,3 +38,37 @@ class HomeViewController: UIViewController {
         
     }
 }
+
+
+extension HomeViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return articlesToDisplay.count
+        
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: ArticleCell.articleCellIdentifier, for: indexPath) as! ArticleCell
+        
+        cell.dateLabel.text = articlesToDisplay[indexPath.row].Date
```

Comment:
It will be more readable if you save the articlesToDisplay[indexPath.row] in a variable at the beginning of the function